draft-cutler-httpbis-partitioned-cookies.md

```diff
@@ -85,8 +88,9 @@ This algorithm could be added after {{Section 5.2 ("Same-site" and "cross-site"
 
 {:quote}
 > 1. Let top-document be the active document in document's browsing context's top-level browsing context.
-> 2. Let "cookie-partition-key" be the site of the top-document when the user agent made the request.
-> 3. If the cookie is being read or written via a "non-HTTP" API, then cookie-partition-key is the site (as defined in {{HTML}}) of the top-document of the document associated with the non-HTTP API.
+> 2. Let "cookie-partition-key" be a tuple whose first element is the site of the top-document when the user agent made the request and whose second element is a boolean (cross-site ancestor) indicating if the cookie was created in a cross-site context as defined in [Section 5.2 of RFC6265bis](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-13#section-5.2).
+> 3. All cookies set on the top-document have a cross-site ancestor value indicating that they were set in a same-site context.
+> 4. If the cookie is being read or written via a "non-HTTP" API, then cookie-partition-key's first element, the site (as defined in {{HTML}}), is the top-document associated with the non-HTTP API. If the origin assocaited with the cookie is not first-party to the site contained in the first element, the second element (cross-site ancestor) must have a value that corresponds to a cross-site context.
```

Comment:
I am not sure this is entirely correct, what if there is a script in an A>B>A embedded frame?

Reply:
Removed the language in lieu of calling out that edge case specifically.